app/Providers/ViewComposerServiceProvider.php

```diff
@@ -31,6 +31,7 @@ public function boot()
     public function register()
     {
         $this->app->view->composer('*', 'CachetHQ\Cachet\Composers\LoggedUserComposer');
+        $this->app->view->composer('*', 'CachetHQ\Cachet\Composers\ThemeComposer');
```

Comment:
Doesn't need to be yet :) In V2 it can be, once we start giving incidents and components their own endpoints.